discussion-questions.md

```diff
@@ -1,8 +1,9 @@
 #  How much time did you spend on this assignment?
 
-_replace this text with your answer_
+Around 5 hours
 
 # Discussion questions / topics
 Below, add one or two things that you might want to discuss when we meet again.
 
-_replace this text with your discussion questions / topics_
\ No newline at end of file
+I think that Scala is very similar to java and python in the way the program runs.
+Also, the importing took a lot of time. 
```

Comment:
What you've written here are true observations, but could benefit from some specificity. In what ways are Scala, Java, and Python similar? By "importing", do you mean how to get Scala up and running (e.g., getting the project into IntelliJ)?

Also, the purpose of the discussion questions / topics is to seed conversation in our group meetings. From your description, it's difficult for me to know what you'd like to focus on in a group session. Would you have wanted to talk about how to import? Would you have wanted to talk about the similarities between Java, Python, and Scala?